client/src/app/my-projects/layout.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+
+import { PropsWithChildren } from "react";
+
+import { MapProvider } from "react-map-gl";
+
+import { LayoutGroup } from "framer-motion";
+
+import MainNav from "@/containers/nav";
+
+import { SidebarProvider } from "@/components/ui/sidebar";
+
+export default function MyProjectsLayout({ children }: PropsWithChildren) {
+  return (
+    <SidebarProvider>
```

Comment:
Please, pull latest changes from `dev`: you don't need to add the SidebarProvider neither the MainNav now. They are part of the root layout now.